molecule/cookiecutter/scenario/verifier/goss/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/verify.yml

```diff
@@ -10,37 +10,49 @@
   vars:
     goss_version: v0.3.6
     goss_arch: amd64
-    goss_dst: /usr/local/bin/goss
-    goss_sha256sum: 53dd1156ab66f2c4275fd847372e6329d895cfb2f0bcbec5f86c1c4df7236dde
-    goss_url: "https://github.com/aelsabbahy/goss/releases/download/{{ goss_version }}/goss-linux-{{ goss_arch }}"
-    goss_test_directory: /tmp
+    goss_bin: /usr/local/bin/goss
+    goss_sha256sum: 53dd1156ab66f2c4275fd847372e6329d895cfb2f0bcbec5f86c1c4df7236dde.
+    goss_test_directory: /tmp/molecule/goss
     goss_format: documentation
   tasks:
     - name: Download and install Goss
       get_url:
-        url: "{{ goss_url }}"
-        dest: "{{ goss_dst }}"
+        url: "https://github.com/aelsabbahy/goss/releases/download/{{ goss_version }}/goss-linux-{{ goss_arch }}"
+        dest: "{{ goss_bin }}"
         sha256sum: "{{ goss_sha256sum }}"
         mode: 0755
-      register: download_goss
-      until: download_goss is succeeded
-      retries: 3
+
+    - name: Create Molecule directory for test files
+      file:
+        path: "{{ goss_test_directory }}"
+        state: directory
+
+    - name: Find Goss tests on localhost
+      shell: "find {{ lookup('env', 'MOLECULE_VERIFIER_TEST_DIRECTORY') }} \\( -name 'test_*.yml' -and -not -name 'test_host_*.yml' \\) -or -name 'test_host_{{ ansible_hostname }}*.yml'"
+      delegate_to: localhost
+      register: test_files
+
+    - name: debug
```

Comment:
Should only be turned on if we're passing in debug arguments?

Reply:
Isn't that what verbosity does? Or do you mean molecule debug arg? 
```
Note
Molecule will remove any options matching ‘^[v]+$’, and pass -vvv to the underlying ansible-galaxy command when executing molecule –debug.
```
So if you pass --debug, verbosity=3 should match this behavior.

Reply:
Cool.